sdk/core/System.Net.ClientModel/tests/OpenAIClient/OpenAIClient.cs

```diff
@@ -1,23 +1,20 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-using Azure.Core;
-using Azure.Core.Pipeline;
-using System;
 using System.Net.ClientModel;
-using System.Threading;
+using System.Net.ClientModel.Core.Pipeline;
 
 namespace OpenAI;
 
 public class OpenAIClient
```

Comment:
We're not using any of these tests today so if you don't need them, feel free to rip them out